### PR TITLE
Issue #49

### PR DESCRIPTION
**Pull request by anuveyatsu**
* refactored `HandsOnTable` component to render different `div` elements depending on resource's index
* refactored `TabularResourceViewContainer` component to work with multiple data resources
* refactored `DataPackageViewContainer` to work with multiple data resources
* refactored `index.js`:
  * moved redux dispatch to here
  * refactored to render multiple containers if there are multiple div elements with vis class

### CODE REVIEW

File: src/containers/DataPackageViewContainer.js [hunk 1]
```diff
@@ -33,8 +27,8 @@ export class DataPackageViewContainer extends React.Component {
 
   convertData(data, dp) {
```
Comment (Fak3) [inline src/containers/DataPackageViewContainer.js:33]: Can you please add some comments, what this method does. And for other methods, as possible please.

Comment (Fak3) [reply src/containers/DataPackageViewContainer.js:33]: And for the component calss too. What it should display.

**Review by Fak3 — COMMENTED** [review]
Btw, is it ok that our jsx files have 'js' extension? Does it solve anything, or can we rename?

**Comment by anuveyatsu** [comment]
@Fak3 JSX code is handled by babel presets. There is no need to rename them to `.jsx`.

File: src/containers/TabularResourceViewContainer.js [hunk 2]
```diff
@@ -14,10 +16,11 @@ export class TabularResourceViewContainer extends React.Component {
     }
   }
 
+  //Takes a single resource and returns Handsontable spec
   buildHandsontableSpec(data) {
     let spec = {
-      data: data.slice(1),
-      colHeaders: data[0],
+      data: data.slice(1), //excluding headers
```
Comment (Fak3) [inline src/containers/TabularResourceViewContainer.js:22]: Minor stuff for the future: it is better to divide trailing comments from the code by 2 spaces at least. This is from pep8, but probably applies to js too.

**Review by Fak3 — APPROVED** [review]
LGTM

**Comment by Fak3** [comment]
@subhankarb can you please review and merge it if all is ok?

**Pull request merged**